SupermarketReceipt-solution/build.gradle

```diff
@@ -8,10 +8,9 @@ repositories {
 }
 
 dependencies {
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
-    testImplementation 'org.junit.jupiter:junit-jupiter-params:5.8.1'
-    testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
-    testImplementation 'com.approvaltests:approvaltests:12.1.1'
+    testImplementation(platform('org.junit:junit-bom:5.9.2'))
```

Comment:
This is inspired by https://github.com/junit-team/junit5-samples/blob/main/junit5-jupiter-starter-gradle/build.gradle

Reply:
I did the same in the other Gradle projects.